### ISSUE TITLE
Clothing listed as (poor fit) in crafting preview.

### ISSUE THREAD

**Issue by Sathra225** [CleverRaven/Cataclysm-DDA#57143]
### Describe the bug

When in the crafting menu, any clothing that is able to be 'refitted' is listed as (poor fit). Doesn't apply to clothing that can't be refitted.
When the item is crafted, it is fitted to the player as previously.
Seems to be a UI bug.

### Steps to reproduce

1. Open crafting menu and tab to armor.
2. Scroll to any non XL/XS clothing item able to be refitted.
3. Crafting preview on right side states it will be (poor fit).

### Expected behavior

Clothing listed in crafting menu previews as already fitted.

### Screenshots

<img width="960" alt="fitted chestwrap" src="https://user-images.githubusercontent.com/100510526/165081446-0a436b5c-6a38-461d-861f-48d2642bd5b5.png">
No false info.
<img width="960" alt="poorfit bikini" src="https://user-images.githubusercontent.com/100510526/165081457-b5a0982d-d05b-4a9b-856e-4291378ee120.png">
False info, crafted item is fitted.

### Versions and configuration

Current experimental at time of posting.

**Comment by ZeroInternalReflection** [comment]
It looks like that's because variable size clothing (flag_VARSIZE) is poor fitting by default unless it's been fitted (flag_FIT).  The crafting menu just looks at the default version.  It's fairly simple to add flag_FIT when building the menu, resulting in this:
![Fitted_Bikini](https://user-images.githubusercontent.com/89038572/165148700-f22d573b-5329-4eee-9dc9-9d5dba735e84.png)

The tricks are:
1. It is apparently possible to craft poorly-fitted clothing.  If you make clothing from other poorly-fitted clothing, it will remain poorly-fitted.
2. Trying to have the crafting menu figure out whether the result would be poorly-fitted or not before you start crafting would probably be quite difficult to implement right now
3. The simple fix I added above might break some other recipe displays.  A fair bit of testing will be required

What would make sense to me in the final condition:
1. The crafting result in this screen is always shown as properly-fitted.
2. When crafting starts, if it's going to result in a poorly-fitted item, a popup shows up to ask if the player wants to continue

**Comment by Termineitor244** [comment]
@Jan-Blasiak It seems you still need to be added to the allowed users for the bot.

/confirmed

**Comment by casswedson** [comment]
Nah, I don't think so it's more my bad for not looking into the username beyond @-ing them in a comment
At this point we just need a pr adding Jan, I wanted to make it, but I also wanted to ask you to maybe look into adding more people to the list (@ me in discord if you have a response)

**Comment by Jan-Blasiak** [comment]
D:
